src/torchcodec/decoders/_video_decoder.py

```diff
@@ -379,3 +416,58 @@ def _get_and_validate_stream_metadata(
         end_stream_seconds,
         num_frames,
     )
+
+
+def read_custom_frame_mappings(
```

Comment:
Nit

```suggestion
def _read_custom_frame_mappings(
```

This makes it more obvious that it is private. It is already private even without the leading underscore, because it's only exposed within `_video_decoder.py` which itself has the underscore. But explicitly having an underscore makes it even more obvious, and consistent with the other private functions defined in this file